YACS-RCOS/yacs.n
# Update ColorService.js

Color Service

> Closes #58

Added 5 more colors to ColorService.js, teal, maroon, pink, dark blue, and dark green.

**Test Procedure**

Add classes to the course scheduler, and you will be able to see the new colors added.

**Photos**

<img width="656" alt="updated_colors" src="https://user-images.githubusercontent.com/78986694/110714571-ab7a5080-81d1-11eb-9b5d-c340586a0ff5.PNG">

**Additional Info**

Do not take 36 credit hours. You will die.
-Nick King and Joshua Marion
